lib/generators/boring/rack_mini_profiler/install/install_generator.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module Boring
+  module RackMiniProfiler
+    class InstallGenerator < Rails::Generators::Base
+      desc 'Adds rack mini profiler to the application'
+
+      def add_rack_mini_profiler_gem
+        say 'Adding rack mini profiler gem', :green
+
+        gem_content = <<~RUBY
```

Comment:
I think we can add gem normally instead of injecting into file. any reason you are doing in this way?